brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
 // A default value for the "More information about this update" link
 // supplied in the "An update is available" page of the update wizard.
 pref("app.update.url.details", "https://www.ghostery.com/%LOCALE%/browser/notes");
-
-// Disable System Addon updates
-pref("extensions.systemAddon.update.url", "");
-
 // The number of days a binary is permitted to be old
 // without checking for an update.  This assumes that
 // app.update.checkInstallTime is true.
 pref("app.update.checkInstallTime.days", 63);
-
 // Give the user x seconds to reboot before showing a badge on the hamburger
 // button. default=immediately
 pref("app.update.badgeWaitTime", 0);
 
-// Number of usages of the web console.
-// If this is less than 5, then pasting code into the web console is disabled
-pref("devtools.selfxss.count", 5);
```

Comment:
not sure why this was set like this, better to keep it as it was for now

Reply:
I'm unsure why this is needed as well. It is already the default behavior in Firefox releases.

In the Betterfox.js draft, @2br-2b [talked about](https://github.com/ghostery/user-agent-desktop/pull/484#issuecomment-777484265) removing prefs that are already the default in Firefox.

So the bigger question is: Are we definitely removing prefs that are already the default in Firefox releases?

Reply:
Just wanted to comment that I don't currently work at Ghostery, so take everything I say with a grain of salt :-)

I would think not including default preferences would ever-so-slightly speed up the browser, plus it might help to avoid confusion in some places, but I don't think it matters that much. I just think this redundancy is unnecessary, that's all :-)

Reply:
If that's the current default, then sure, please go ahead and remove it. It make sense! Thank you both!

Reply:
@2br-2b, thank you for clarifying your role as I was not aware lol.

- Speed isn't a concern here IMO as the redundant code is very small.
- Avoiding confusion is a priority for me personally as this project is much larger than my `user.js` lol. I'm sure the other devs would be appreciative.
- I'm a big believer in efficiency. Minimizing code redundancy would limit any potential confusion for the other devs as well.

I'll remove default prefs moving forward and add some kind of "already a default" comment as an explanation.